From 47a523470e08a719f165549f59cd78baef4c7540 Mon Sep 17 00:00:00 2001
From: Sven-Hendrik Haase <svenstaro@gmail.com>
Date: Fri, 5 Jun 2020 08:54:48 +0200
Subject: [PATCH] Use DESTDIR instead of uncommon DISTDIR

It's quite common to see DESTDIR being used with Makefiles but DISTDIR is very uncommon.
---
 Makefile | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/Makefile b/Makefile
index 4d9948adeec..d213abdc27d 100644
--- a/Makefile
+++ b/Makefile
@@ -1,17 +1,17 @@
 VERSION=$(shell cat emscripten-version.txt | sed s/\"//g)
-DISTDIR=../emscripten-$(VERSION)
+DESTDIR=../emscripten-$(VERSION)
 DISTFILE=emscripten-$(VERSION).tar.bz2
 
 dist: $(DISTFILE)
 
 install:
-	@rm -rf $(DISTDIR)
-	./tools/install.py $(DISTDIR)
+	@rm -rf $(DESTDIR)
+	./tools/install.py $(DESTDIR)
 
 # Create an distributable archive of emscripten suitable for use
 # by end users.  This archive excludes parts of the codebase that
 # are you only used by emscripten developers.
 $(DISTFILE): install
-	tar cf $@ $(EXCLUDE_PATTERN) -C `dirname $(DISTDIR)` `basename $(DISTDIR)`
+	tar cf $@ $(EXCLUDE_PATTERN) -C `dirname $(DESTDIR)` `basename $(DESTDIR)`
 
 .PHONY: dist install
